docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - kibana
 
   api:
-    build: ./lab/api
+    image:  ubiqube/msa2:api-2.1.0rc1
```

Comment:
@n1654 
I think that with this line, the Dockerfile under lab/api won't be taken into account ?

Reply:
@abr-ubiqube you are right,
for v3 there is the only option to run both environments with docker-compose command:
`$ docker-compose -f ./docker-compose.yml -f ./lab/sdwan/docker-compose.yml up -d`

and for the lab, context should be changed to
`context: ./lab/sdwan`
instead of
`context: .`

I suggest to run lab independently thus we can create multiple lab environments that perfectly fit customer interests.

Reply:
@abr-ubiqube please don't mind previous comment....

to make build from lab/api
here is an example:
 `build:
      context: ./lab/api
      dockerfile: Dockerfile`